### PR TITLE
pt_BR Fixes and new translations

### PR DESCRIPTION
**Pull request by brunowowk**

### CODE REVIEW

**Comment by kajmagnus** [comment]
Hi Bruno, thanks for the translations and fixes. Could you please squash the commits, and change the *commit message* so it ends with "I, Bruno ....  agree to the ..."  ?

(It's the *commit message* that should include that text, not the GitHub issue title :- ))

File: translations/pt_BR/i18n.ts [hunk 1]
```diff
@@ -637,7 +637,7 @@ var t_pt_BR: TalkyardTranslations = t = {
     ClickUnaccept: "Clique para desafazer a aceitação desta resposta",
     PostAccepted: "Este post foi aceitado como resposta",
 
-    NumLikes: (num: number) => num === 1 ? "1 Curte" : num + " Curtem",
+    NumLikes: (num: number) => num === 1 ? "1 Curtiu" : num + " Curtiram",
```
Comment (kajmagnus) [inline translations/pt_BR/i18n.ts:640]: Are bot `Curte` and `Curtiu` Brazilian Portuguese?  (none of them Portuguese Portuguese?)

I'm curious about what's the difference :- )

File: translations/pt_BR/i18n.ts [hunk 2]
```diff
@@ -526,21 +526,21 @@ var t_pt_BR: TalkyardTranslations = t = {
...
+    StatusPlanned: "Planejado",
     TooltipProblPlanned: "Estamos planejando consertar isto",
     TooltipIdeaPlanned: "Estamos planejando implementar isto",
 
-    StatusNew: "New",  // MISSING
-    StatusNewDtl: "New topic, under discussion",  // MISSING
-    TooltipUnsProbl: "Isto é um problema não resolvido",
+    StatusNew: "Novo",
+    StatusNewDtl: "Novo tópico, em discussão",
+    TooltipUnsProbl: "Isso é um problema sem solução",
```
Comment (kajmagnus) [inline translations/pt_BR/i18n.ts:543]: Are bot `Isto é um problema não resolvido` and `Novo tópico, em discussão` Brazilian Portuguese?  (none of them Portuguese Portuguese?)

I'm curious about what's the difference :- )  Anyway, `sem solução` is a bit shorter, that's nice

File: translations/pt_BR/i18n.ts (hunk 1 above)
Comment (brunowowk) [reply translations/pt_BR/i18n.ts:640]: Yeah, both are correct in brazilian portuguese.
Curte (in the present) is also correct, but "Curtiu" (in the past) is the most used form on social networks, for instance in facebook:
_2__Facebook

**Pull request closed without merging**